lang/en/tool_excimer.php

```diff
@@ -83,3 +84,10 @@
 $string['reason_auto'] = 'Auto';
 $string['reason_flameall'] = 'Flame All';
 
+// Miscellaneous.
+$string['deleteall'] = 'Delete All';
+$string['deleteallwarning'] = 'This will remove ALL stored profiles. Continue?';
+$string['deleteprofile'] = 'Delete profile';
+$string['deleteprofilewarning'] = 'This will remove the profile. Continue?';
+$string['allprofiesdeleted'] = 'All profiles have been deleted';
+$string['profiedeleted'] = 'Profile Deleted';
```

Comment:
Typically all sentences should end with a fullstop. Please add these.

(This does not need to be added for action buttons)